src/common/perf_histogram.h

```diff
@@ -76,8 +76,10 @@ class PerfHistogram : public PerfHistogramCommon {
   }
 
   /// Copy from other histogram object
-  PerfHistogram(const PerfHistogram &other)
-      : m_axes_config(other.m_axes_config) {
+  PerfHistogram(const PerfHistogram &other) {
+    for (int i = 0; i < DIM; i++) {
```

Comment:
@tchaikov Great idea! I guess that's what std::array was created for. And I'm almost sure that copying such array to another one would be automatically converted to memcpy if the element type is trivially copyable.

Will give it a try in a second.